recipe/yum_requirements.txt

```diff
@@ -1,5 +1,2 @@
-libibcm-devel
-libibverbs-devel
-librdmacm-devel
 numactl-devel
 rdma-core-devel
```

Comment:
From installing with `yum` we can see the other ones are picked up (`ibcm` was dropped in a older version of rdma-core, please see PR ( https://github.com/linux-rdma/rdma-core/pull/279 ) for context):

<details>

```
Dependencies Resolved

================================================================================
 Package                    Arch       Version                Repository   Size
================================================================================
Installing:
 numactl-devel              x86_64     2.0.12-5.el7           base         24 k
 rdma-core-devel            x86_64     22.4-6.el7_9           updates     254 k
Installing for dependencies:
 groff-base                 x86_64     1.22.2-8.el7           base        942 k
 hwdata                     x86_64     0.252-9.7.el7          base        2.5 M
 ibacm                      x86_64     22.4-6.el7_9           updates      83 k
 initscripts                x86_64     9.49.53-1.el7_9.1      updates     440 k
 iproute                    x86_64     4.11.0-30.el7          base        805 k
 iptables                   x86_64     1.4.21-35.el7          base        432 k
 libibumad                  x86_64     22.4-6.el7_9           updates      25 k
 libibverbs                 x86_64     22.4-6.el7_9           updates     269 k
 libmnl                     x86_64     1.0.3-7.el7            base         23 k
 libnetfilter_conntrack     x86_64     1.0.6-1.el7_3          base         55 k
 libnfnetlink               x86_64     1.0.1-4.el7            base         26 k
 libnl3                     x86_64     3.2.28-4.el7           base        278 k
 librdmacm                  x86_64     22.4-6.el7_9           updates      64 k
 numactl-libs               x86_64     2.0.12-5.el7           base         30 k
 pciutils                   x86_64     3.5.1-3.el7            base         93 k
 pciutils-libs              x86_64     3.5.1-3.el7            base         46 k
 perl                       x86_64     4:5.16.3-299.el7_9     updates     8.0 M
 perl-Carp                  noarch     1.26-244.el7           base         19 k
 perl-Encode                x86_64     2.51-7.el7             base        1.5 M
 perl-Exporter              noarch     5.68-3.el7             base         28 k
 perl-File-Path             noarch     2.09-2.el7             base         26 k
 perl-File-Temp             noarch     0.23.01-3.el7          base         56 k
 perl-Filter                x86_64     1.49-3.el7             base         76 k
 perl-Getopt-Long           noarch     2.40-3.el7             base         56 k
 perl-HTTP-Tiny             noarch     0.033-3.el7            base         38 k
 perl-PathTools             x86_64     3.40-5.el7             base         82 k
 perl-Pod-Escapes           noarch     1:1.04-299.el7_9       updates      52 k
 perl-Pod-Perldoc           noarch     3.20-4.el7             base         87 k
 perl-Pod-Simple            noarch     1:3.28-4.el7           base        216 k
 perl-Pod-Usage             noarch     1.63-3.el7             base         27 k
 perl-Scalar-List-Utils     x86_64     1.27-248.el7           base         36 k
 perl-Socket                x86_64     2.010-5.el7            base         49 k
 perl-Storable              x86_64     2.45-3.el7             base         77 k
 perl-Text-ParseWords       noarch     3.29-4.el7             base         14 k
 perl-Time-HiRes            x86_64     4:1.9725-3.el7         base         45 k
 perl-Time-Local            noarch     1.2300-2.el7           base         24 k
 perl-constant              noarch     1.27-2.el7             base         19 k
 perl-libs                  x86_64     4:5.16.3-299.el7_9     updates     690 k
 perl-macros                x86_64     4:5.16.3-299.el7_9     updates      44 k
 perl-parent                noarch     1:0.225-244.el7        base         12 k
 perl-podlators             noarch     2.5.1-3.el7            base        112 k
 perl-threads               x86_64     1.87-4.el7             base         49 k
 perl-threads-shared        x86_64     1.43-6.el7             base         39 k
 rdma-core                  x86_64     22.4-6.el7_9           updates      51 k
 sysvinit-tools             x86_64     2.88-14.dsf.el7        base         63 k

Transaction Summary
================================================================================
Install  2 Packages (+45 Dependent packages)
```

</details>